### ISSUE TITLE
Integration with action launcher

### ISSUE THREAD

**Issue by arihantanchalia** [BranchMetrics/sesame_issues#77]
Please add integration to Action Launcher.

**Comment by philwall2** [comment]
Hi. We'd love to do this and have been in conversations with the Action Dev. We provided him with everything he needs from Sesame. But he has yet to follow through. If you'd like, it might help if you reach out to him and ask for the feature.

**Comment by philwall2** [comment]
@arihantanchalia any word back from Action? We still haven't heard back from them. I'm curious if they want to do this.

**Comment by arihantanchalia** [comment]
Well, this was their reply :
"We'll give it some thought, and evaluate how much work would be involved, whether we agree its a good fit for Action Launcher, and how it aligns with the current plans for future Action Launcher updates."

**Comment by philwall2** [comment]
Hmm, well that's not much of a positive signal. Thanks for asking. FWIW, we're ready to go on our end.

**Comment by CCninja86** [comment]
How does the integration with Nova work? I notice the integration with Nova is more...integrated. Like, the results appear in their default search box, but with Hyperion and Lawnchair it basically just launches the app when you tap the search box. Is this a thing done by the developers or does Nova have a different kind of integration?

**Comment by philwall2** [comment]
We have 3 options: 

1. Access our Search data and do with it what you want. Like integrate it into your Native Search like Nova did
2. Access long press data only, show it when users long press an app icon (Lawnchair and Hyperion)
3. Open the Sesame widget (Hyperion and Lawnchair)

To make things more complicated... even though Nova is the only one who did the full integration, they are actually using a much older version of our data library and it's missing several features that Sesame Search now supports.

**Comment by skittles9823** [comment]
@philwall2 does the current lib (that lc and hyperion use) support accessing the search data like nova does, but obviously more updated?

**Comment by philwall2** [comment]
@skittles9823 yes. LMK if you've got more questions. Steve and I can show you where etc.

**Comment by skittles9823** [comment]
@philwall2 ahh cool. I'll have to ask Till and Anas to see if they can incorporate it into search when they have some free time.

Pretty sure the Hyperion team is waiting for Paphon to start the Android 10 merge before they do anything though.
